hack/arktos-worker-up.sh

```diff
@@ -40,5 +63,8 @@ sudo ./hyperkube kubelet \
 --cluster-domain=cluster.local \
 --container-runtime-endpoint="containerRuntime,container,/run/containerd/containerd.sock;vmRuntime,vm,/run/virtlet.sock" \
 --runtime-request-timeout=2m \
---port=10250
+--port=10250 \
+> /tmp/kubelet.worker.log 2>&1 &
```

Comment:
nit: maybe use a randomly named file to prevent accidentally overwriting user's file with this same name.

Reply:
Good point in general. But considering other arktos components are using the fixed named log files(like kube-apiserver.log); it is better to follow the suit in this case.